contracts/Bridge/XVSBridgeAdmin.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+pragma solidity 0.8.13;
+
+import { AccessControlledV8 } from "@venusprotocol/governance-contracts/contracts/Governance/AccessControlledV8.sol";
+
+interface IXVSBridge {
+    function transferOwnership(address addr) external;
+}
+
+contract XVSBridgeAdmin is AccessControlledV8 {
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    IXVSBridge public immutable XVSBridge;
+
+    mapping(bytes4 => string) public functionRegistry;
+
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor(address XVSBridge_) {
+        require(XVSBridge_ != address(0), "Address must not be zero");
+        XVSBridge = IXVSBridge(XVSBridge_);
+        _disableInitializers();
+    }
+
+    /**
+     * @notice Invoked when called function does not exist in the contract
+     */
+    fallback() external {
+        string memory res = _getFunctionName(msg.sig);
+        require(bytes(res).length != 0, "Function not found");
+        _checkAccessAllowed(res);
+        (bool ok, ) = address(XVSBridge).call(msg.data);
+        require(ok, "call failed");
+    }
+
+    function initialize(address accessControlManager_) external initializer {
+        require(address(accessControlManager_) != address(0), "Address must not be zero");
+        __AccessControlled_init(accessControlManager_);
+
+        functionRegistry[hex"7adbf973"] = "setOracle(address)";
```

Comment:
Would it make sense to have a function to upsert and delete entries in the `functionRegistry` map, protected with the `onlyOwner` modifier?

Something like this for the upsert case:
```
function upsert(string calldata signature) public onlyOwner {
  functionRegistry[bytes4(keccak256(bytes(signature)))] = signature;
  // emit event
}
```
This way we could configure the contract from the VIP where we'll own the deployed contract, and we can configure it in the future if needed.

Reply:
Fixed here [f719f81](https://github.com/VenusProtocol/isolated-pools/pull/291/commits/f719f814d46054f22ee9667bf3c47caa53c61fce)